sda-commons-server-auth-testing/src/main/java/org/sdase/commons/server/auth/testing/AuthExtension.java

```diff
@@ -0,0 +1,190 @@
+package org.sdase.commons.server.auth.testing;
+
+import static java.util.Collections.singletonList;
+import static org.junit.Assert.fail;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.Configuration;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.sdase.commons.server.auth.config.AuthConfig;
+import org.sdase.commons.server.testing.Environment;
+
+public class AuthExtension extends AbstractAuth implements BeforeAllCallback, AfterAllCallback {
```

Comment:
Do we want a common naming convention? According to Wiremock we could change the name to "AuthClassExtension".